components/DestinationItem.js

```diff
@@ -5,11 +5,18 @@ export default function DestinationItem({ name }) {
   return (
     <ListBox bulletCircle={bulletCircle.src}>
       <BulletCircle src={bulletCircle.src} height={15} />
```

Comment:
why is the 15 in { }?

Reply:
opps that shouldnt be there. deleted it :)